features/step_definitions/steps.rb

```diff
@@ -32,5 +32,9 @@ def path(pagename)
       new_user_registration_path
     when 'create menu'
       new_menu_path
+    when 'restaurant edit details'
+      edit_restaurant_path(@restaurant)
+    else
+      you_have_not_defined_a_path
```

Comment:
In this case I would return a string. `you_have_not_defined_a_path` will look for a variable or method.